libraries/opensk/src/ctap/client_pin.rs

```diff
@@ -119,6 +119,7 @@ pub enum PinPermission {
     CredentialManagement = 0x04,
     _BioEnrollment = 0x08,
     LargeBlobWrite = 0x10,
+    #[cfg(any(feature = "config_command", feature = "std"))]
```

Comment:
What's the rational to have `std`? If we want `std` to imply `config_command` we can do it at the feature level by adding `config_command` in the dependencies of `std`.

Reply:
An integration test for enterprise attestation (with a customization that allows it) would first need to turn EP on through Config, then call MakeCredential. What the feature implies is that if you use it, your Env turns on EP through some other means. We should probably also do that ourselves, now that you say it.

Not allowing the Config command in `std`. would imply that we can't use our current `test_helpers` and would need to add a lot more conditionals throughout tests.

Reply:
> Not allowing the Config command in std

What do you mean? I said the opposite: make the `config_command` a dependency of `std`, i.e. the line that currently looks like:

```toml
# libraries/opensk/Cargo.toml
std = ["crypto/std", "persistent_store/std", "rand/std_rng"]
```

would become:

```toml
std = ["crypto/std", "persistent_store/std", "rand/std_rng", "config_command"]
```

Reply:
I didn't want to imply that this isn't a solution. Just stating the requirements. I agree this is nicer already.

Reply:
Done